bundles/amazon/ack/elasticache/bundle/config/00-schema.yml

```diff
@@ -0,0 +1,63 @@
+#@data/values-schema
+---
+#@schema/title "Name"
+#@schema/desc "Name of the ElastiCache instance"
+name: ""
+
+#@schema/title "Namespace"
+#@schema/desc "Namespace to deploy the kubernetes resources to"
+namespace: ""
+
+#@schema/title "CreateNamespace"
+#@schema/desc "Whether to create the namespace for the resources or not"
+createNamespace: False
+
+#@schema/title "CacheSubnetGroupName"
+#@schema/desc "Name of the cache subnet group to create the replication group in. It has to be available upfront."
+cacheSubnetGroupName: ""
+
+#@schema/title "CacheNodeType"
+#@schema/desc "Type of nodes used for the replication group"
+cacheNodeType: "cache.t2.micro"
+
+#@schema/title "VpcSecurityGroupIDs"
+#@schema/desc "The list of security groups to associate to the replication group"
+vpcSecurityGroupIDs:
+  - ""
+
+#@schema/title "EngineVersion"
+#@schema/desc "The version of the Redis engine to use. Available versions can be obtained running `aws elasticache describe-cache-engine-versions --engine redis`."
+#@schema/default "6.2"
```

Comment:
**question** Curious why we use `#@schema/default` here over setting the value? It seems like `cacheNodeType` does the opposite.

Reply:
You're right, we don't need `#@schema/default` annotations for strings, numbers and booleans